docs/Building-Transmission.md

```diff
@@ -39,20 +39,52 @@ cmake --build build -t transmission-gtk
 ## On Unix ##
 ### Prerequisites ###
 
-#### Debian 11 and Newer ####
+#### Debian 12 / Bookworm ####
+On Debian, you can build transmission with a few dependencies on top of a base installation.
+
+For building transmission-daemon you will need basic dependencies:
+```bash
+$ sudo apt install build-essential cmake git libcurl4-openssl-dev libssl-dev
+```
+These packages are not mandatory for a working binary. Transmission brings its own libraries if they aren't installed.
```

Comment:
Not true in case of `libsystemd-dev` which isn't bundled. Although it's true that it's optional, what omitting this package leads to I'm not entirely sure as I'm not using systemd.

Reply:
CMakeLists.txt description for systemd says:
> Add support for systemd startup notification (on systems that support it)

Tracing its usage to daemon/daemon.cc shows me that systemd is being used for `sd_notify` and `sd_notifyf` functions, which are both standard way for informing systemd about changed daemon state.
An example use is:
`sd_notify(0, "STATUS=Closing transmission session...\n");`

You're still correct, transmission does not bring its own systemd library :smile: